scripts/prepare/bundle.ts

```diff
@@ -117,11 +120,24 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
         platform: platform || 'browser',
         external: externals,
 
+        esbuildPlugins: [
+          esbuildAliasPlugin({
+            lodash: 'lodash-es',
+          }),
+          lodashTransformer({
```

Comment:
Why do we need this plugin? Is tree-shaking not correctly working for lodash-es? Or do we want to accommodate all `lodash` usages, which we haven't swapped out by `lodash-es`?